common/persistence/src/main/resources/derivation-maps.yaml

```diff
@@ -274,6 +272,7 @@ tek-field-derivations:
     1797: 2
     1798: 2
     1799: 2
+    1800: 1
```

Comment:
@mlenkeit what do you think, shall we remove the mappings to 1, since it's the default